Selektric/native_mocks.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2022 Bloomberg Finance L.P.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <time.h>
+#include <jni.h>
+#include <string.h>
+#include <sys/types.h>
+
+static jint android_util_Log_println_native(
+    JNIEnv* env,
+    jobject clazz,
+    jint bufID,
+    jint priority,
+    jstring tagObj,
+    jstring msgObj
+) {
+    return 0;
+}
+
+static jstring SystemProperties_get(
```

Comment:
do we still need this one ?

Reply:
yep - I replayed to sanity check, adding each to fix the tests.